pmoves/docs/handoffs/SECRET_ROTATION_RUNBOOK.md

```
@@ -0,0 +1,40 @@
+# Secret Rotation Runbook — `make secrets-rotate`
+
+`make -C pmoves secrets-rotate KEY=<env.shared key> [VALUE=<minted>] [LEN=48]` is the Known Road for rotating a single secret. It surgically replaces one `KEY=` line in `env.shared` (generates a `random_urlsafe` value when `VALUE` is omitted; refuses multi-line values), then runs `chit-export` (encode → CGP bundle) + `secrets-funnel` (propagate to tier files). It does **not** auto-restart services — it prints the remaining steps.
+
+Rotation makes the git-history value **inert** (that's the fix); history-purge is optional. `bootstrap_env.py` is chitBypass-allowed, so this runs as an elevated agent command — no hand-editing of `env.shared`.
+
+> Generated values use `token_urlsafe` → only `-`/`_` (no `/`/`+`), avoiding connection-string URL-encode fragility.
+
+---
+
+## 1. Render webhook shared secret (internal HMAC)
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)